manifests/init.pp

```diff
@@ -105,7 +125,7 @@
 
     package {'influxdb2':
```

Comment:
There were some errors in the acceptance tests related to the ssl files being applied before the `/etc/influxdb` directory exists, and `influxdb.service.d` being applied before the `influxdb` user exists.  I don't know why this happens on Ubuntu and not elsewhere, but I tested this change and it worked.

```
    $package_before = if $use_ssl and $manage_ssl {
      [
        File['/etc/influxdb/cert.pem', '/etc/influxdb/key.pem', '/etc/influxdb/ca.pem', '/etc/systemd/system/influxdb.service.d'],
        Service['influxdb']
      ]
    }
    else {
      Service['influxdb']
    }

    package {'influxdb2':
      ensure  => $version,
      require => $package_require,
      before  => $package_before,
    }
```

Reply:
Hi @SimonHoenscheid, we're still seeing these failures in acceptance tests.

```
Error: Could not set 'file' on ensure: No such file or directory - A directory component in /etc/influxdb/cert.pem20220913-19386-1aod2be.lock does not exist or is a dangling symbolic link (file: /etc/puppetlabs/code/environments/production/modules/influxdb/manifests/init.pp, line: 200)
```

Can you try adding this code so the package is installed before we try to set up the certs?